zerver/tests/test_subs.py

```diff
@@ -59,27 +59,42 @@ class TestCreateStreams(ZulipTestCase):
     def test_creating_streams(self):
         # type: () -> None
         stream_names = [u'new1', u'new2', u'new3']
+        stream_descriptions = [u'des1', u'des2', u'des3']
         realm = get_realm_by_string_id('zulip')
 
         new_streams, existing_streams = create_streams_if_needed(
             realm,
-            [{"name": stream_name, "invite_only": True} for stream_name in stream_names])
+            [{"name": stream_name,
+              "description": stream_description,
+              "invite_only": True}
+             for (stream_name, stream_description) in zip(stream_names, stream_descriptions)])
```

Comment:
The static analysis test failure is because `zip` differs in semantics between Python 2 and Python 3; the fix is to add `from six.moves import zip` at the top of the file.  You can read about the six module if you're curious to learn more about this issue :).